frontend/src/tests/instructor-permission-test.js

```diff
@@ -46,6 +46,63 @@ export function instructorsPermissionTests(api) {
         expect(respSwitchBackUser).toHaveStatus("success");
     }
 
+    /**
+     * Updates a position to include instructor with <instructorId> and
+     * creates a DDAH for one of the assignments realted to that
+     * instructor.
+     *
+     * @param instructorId: int - the unique Id of an instructor
+     *
+     * @returns {Promise<Ddah>}
+     */
+    async function createDDAH(instructorId) {
+        // We first need to update position to include our instructor
+        const existingPosition = databaseSeeder.seededData.positions[0];
+        const updatedPosition = {
+            ...existingPosition,
+            instructor_ids: [...existingPosition.instructor_ids, instructorId],
+        };
+        const positionResponse = await apiPOST(
+            `/admin/positions`,
+            updatedPosition
+        );
+        expect(positionResponse).toHaveStatus("success");
+
+        // We then proceed to create a DDAH for that position
+        // Switch to instructor user so we only have assignments for that instructor
+        await switchToInstructorOnlyUser();
```

Comment:
I think it's confusing to switch users in a utility function. Utility functions should have minimal side effects.

Reply:
We switch back afterwards though, you still think it's not appropriate? If that's the case, what is a way to get instructor-related assignments?

Reply:
You know exactly what the mock data is. You can rely on it.